frontend/src/metabase/static-viz/lib/text.ts

```diff
@@ -1,32 +1,47 @@
 const CHAR_ELLIPSES = "…";
+const DEFAULT_FONT_WEIGHT = 400;
 
 // TODO: Replace this rough simple approximation with a correct one
-const getCharWidth = (fontSize: number) => {
+const getCharWidth = (fontSize: number, fontWeight: number) => {
```

Comment:
here's a possible thing we might be able to do: there's an npm thing for using python fonttools in JS, which actually goes and reads off a ttf file and then measures widths in ems and gives them to you.

https://www.npmjs.com/package/fonttools

this might still have the overall problems of FFI being a terrible idea. another thing that could be done is just using FontTools on Lato until we have a table that we just copy-and-paste and getCharWidth takes from the table...